src/adapters/refract-adapter.coffee

```diff
@@ -0,0 +1,45 @@
+_ = require('./refract/helper')
+blueprintApi = require('../blueprint-api')
+
+getDescription = require('./refract/getDescription')
+transformSections = require('./refract/transformSections')
+
+transformAst = (element) ->
+
+  applicationAst = new blueprintApi.Blueprint({
+    name: _.chain(element).get('meta.title', '').fixNewLines().value()
+    version: '22' # how about this? we don't have in refract parse result at all
```

Comment:
I think we did not. But it should be probably same as is here: https://github.com/apiaryio/metamorphoses/blob/shared/refract/src/blueprint-api.coffee#L7